blogposts/2020/announcing-sourcegraph-3.16.md

```diff
@@ -0,0 +1,238 @@
+
+Campaign changeset usage counts including changesets created, added, and merged will be sent back in pings.
+
+## Best in class syntax highlighting
+
+TODO: Erik is creating screenshots
+
+We have made major improvements to syntax highlighting throughout Sourcegraph. In addition to general improvements, six new languages are supported (Smarty, Ethereum / Solidity / Vyper, Cuda, COBOL, vb.NET, and ASP.NET), and 30 new file extensions are now detected. 
+
+Additionally, this release adds best-in-class syntax highlighting to code diffs. This richer view of diffs can easily be accessed from your pull request (or merge request) using the Sourcegraph browser extension by clicking on the Sourcegraph button in a file header. Alternatively, you can compare revisions directly from Sourcegraph by going to the compare section from a repository page. You can also see this in action on your [campaigns](#campaigns) to make changes across all the repositories at your organization. 
+
+Take a look at the syntax highlighting on a diff that &lt;WHAT IS THIS DIFF ABOUT> in the Sourcegraph repository.
+
+## Easier to read queries in plain text mode
+
+TODO: video
+
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://www.youtube.com/embed/{VIDEO_ID}" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://youtu.be/{VIDEO_ID}" target="_blank">View on YouTube</a></p>
+</p>
+-->
+
+We introduced the smart search bar for plain text mode that provides syntax highlighting, hover tooltips, and diagnostics on search queries in [Sourcegraph 3.13](https://about.sourcegraph.com/blog/sourcegraph-3.13#smart-search-field-in-plain-text-mode). This means that Sourcegraph can provide useful hints, such as fixing invalid quotes, to get to search results faster. Sourcegraph 3.16 removes the feature flag and makes this the standard experience for all users in plain text mode.
+
+## Performance improvements for precise code intelligence
+
+![performance-improvements-code-intel](/blog/3-16-ts-to-go-lsif.png "performance-improvements-code-intel")
+
+As of Sourcegraph 3.16, our precise code intelligence backend has been rewritten from TypeScript to Go. This effort is part of a larger effort to aggressively optimize conversion and querying of LSIF data, which will start becoming more apparent in upcoming releases. Even with this initial improvement, we’re seeing improvements on precise code intelligence query performance. The following graph shows the five-run average duration of our [integration test](https://github.com/sourcegraph/sourcegraph/tree/5f51043ad2130a1acdcfca8b969f907cd03a220d/internal/cmd/precise-code-intel-test) before and after the rewrite.
+
+## Discover useful scopes with `repogroup` autocompletion
+
+TODO: video
```

Comment:
```suggestion
<img src="https://storage.googleapis.com/sourcegraph-assets/repogroup_completion.gif" alt="Repogroup autocompletion">
```

@christinaforney this is a simple change, so I think a video with voiceover would be overkill here. I put together a gif instead.

Reply:
Nice! Thank you!